src/models/Collection.ts

```diff
@@ -73,7 +73,7 @@ export const CollectionModel = {
       _id: ObjectId(),
       url: input.url,
       label: input.label,
-      cmsId: input.cmsId,
+      cmsId: input.cmsId || input.id, // If collection is imported directly from CMS, we need to use id
```

Comment:
This was confusing to me so I investigated - it looks like `Collection.addOne` is used in three cases:
- Creating a new Collection from My Space, in which case it is always initialized with an empty array of bookmarks
- Adding a Bookmark to a new Collection from S&A (alpha sort), in which case it is always creating a new Collection with a single Bookmark from the CMS. It looks like here, the bookmark ID is mapped to `cmsId` in the input at the UI layer.
- Adding the example collection to a new user (below), where bookmarks are inserted directly from the CMS.

I think we should make all use of this operation consistent, so that it's either always `cmsId: input.cmsId` or `cmsId: input.id`. that change can happen either when the example collection is inserted, or we can modify the S&A operation to pass bookmark `id` and let it map to `cmsId` at the database layer. I don't have a strong opinion about which one we go with.